js/coinmate.js

```diff
@@ -27,7 +27,7 @@ module.exports = class coinmate extends Exchange {
             },
             'urls': {
                 'logo': 'https://user-images.githubusercontent.com/1294454/27811229-c1efb510-606c-11e7-9a36-84ba2ce412d8.jpg',
-                'api': 'https://coinmate.io/api',
+                'api': 'https://coinmate.docs.apiary.io',
```

Comment:
I think this is a wrong key to edit the doc url )

Reply:
See line 34 below.

Reply:
https://coinmate.io/api just 404s

Reply:
Wrong key?

Reply:
Oh i see, silly me. thanks